Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -569,3 +569,139 @@
   - type: GhostTakeoverAvailable
   - type: Loadout
     prototypes: [ BoxingKangarooGear ]
+
+- type: entity
+  name: Smile
+  id: MobSlimesPet
+  parent: SimpleMobBase
```

Comment:
parent from slimes instead, i see a lot that could be de-duplicated like this

Reply:
Done, won a few lines. Slime has too many minor changes for the sake of survivability.